<launch>

  <include file="$(find unsw_vision_utils)/launch/pcl-republisher.launch"/>
  <include file="$(find yolov8_ros)/launch/tracker_2d.launch"/>
  
 <node pkg="unsw_vision_utils" type="PositionGenerator" name="PositionGenerator">
    <param name="output_topic" type="str" value="/unsw_vision/detections/objects/positions"/>
    <param name="input_pcd_topic" type="str" value="/points"/>
    <param name="input_obj_topic" type="str" value="/unsw_vision/detections/objects"/>
    <param name="target_frame_id" type="str" value="map"/>
    <param name="source_frame_id" type="str" value="head_rgbd_sensor_rgb_frame"/>
</node>
</launch>
